functions/log/2.0/index.js

```diff
@@ -0,0 +1,15 @@
+const log = async ({ severity, variables }) => {
+  const severityTypes = ['log', 'info', 'debug', 'warn', 'error'];
+
+  if (severityTypes.includes(severity)) {
+    variables.forEach((variable) => {
+      // eslint-disable-next-line no-console
+      console[severity](`${variable.key} : ${JSON.stringify(variable.value)}`);
+    });
+  } else {
+    // eslint-disable-next-line no-console
+    console.error('Unknown severity type');
```

Comment:
I have doubts about this `else` because this will also result in a log message @archan937 what do you think?

Reply:
Yeah. We can leave it out, I guess.